Lib/test/test_annotationlib.py

```diff
@@ -413,19 +416,28 @@ def f2(a: undefined):
             annotationlib.get_annotations(f2, format=Format.FORWARDREF),
             {"a": fwd},
         )
-        self.assertEqual(annotationlib.get_annotations(f2, format=2), {"a": fwd})
+        self.assertEqual(annotationlib.get_annotations(f2, format=3), {"a": fwd})
```

Comment:
Why is `format` hard-coded here?  I'd prefer you think of it like an enum, an arbitrary value whose actual value doesn't matter.  I don't think it's necessary to externally test that the format constants are specific values; they happen to be 1, 2, 3, 4, but I don't want external users to rely on that or ever hard-code them.

Reply:
The intent was to test that the function accepts both the integer value and the enum. If you think it's OK to accept only the enum, I can switch to that model.